packages/http-client/src/index.ts

```diff
@@ -1,80 +1,46 @@
 import fetch from "isomorphic-fetch";
-import type { Includes, Project, Page } from "./index.d";
+import type { BaseInstance, UserProp } from "@webstudio-is/react-sdk";
+import type { Breakpoint } from "@webstudio-is/css-data";
+
+export type Page = {
+  id: string;
+  root: [string, string];
+};
+export type Project = {
+  pages: {
+    [path: string]: {
```

Comment:
ultimately this data structure isn't great I think, ultumately you want pages to be an array:

```
pages: [{path, page, tree, props etc}]
```

Reply:
For me, both are fine. So I will make it ultimate 😎.

Reply:
generally for apis its not great to have collections expressed as hashes. Hashes is an optimization for certain access patterns, that should be done on the consumer site when necessary